pkg/rehearse/metrics.go

```diff
@@ -0,0 +1,131 @@
+package rehearse
+
+import (
+	"encoding/json"
+	"github.com/openshift/ci-operator-prowgen/pkg/config"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/test-infra/prow/apis/prowjobs/v1"
+	prowconfig "k8s.io/test-infra/prow/config"
+)
+
+type ExecutionMetrics struct {
+	SubmittedRehearsals sets.String
```

Comment:
Might seralize better if we make these slices on output

Reply:
Yeah, probably nicer